academic-calendar/react/babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    presets: [
+        '@babel/preset-env',
+        ['@babel/preset-react', { runtime: 'automatic' }],
+    ],
+};
```

Comment:
Add a newline.  Linter probably should have caught this.  Assuming I configured the lint action to look at this file.

Reply:
I saw that the `yarn run lint` script only check the `src` folder. We can change it to something like this:
```
"scripts": {
    ...
    "lint": "eslint src/**/*.jsx src/**/*.js ./*.js __tests__/**/*.js"
  }
```

Basically telling it to also look into `__test__` folder as well as any js file in the project root folder, since that's where the config files are. I can add that in a new commit if you think it looks good.